crpg.py

```diff
@@ -81,19 +83,21 @@ def generate(filename):
         node_mapping: dict[str, Node] = {}
         n_types = {
             "starting": Location,
-            "node": Node,
+            "node": Action,
             "location": Location,
             "fight": Fight,
             "run": Run
         }
 
         c_funcs: dict[str, Callable[[Game, Node, Node], None]] = {
             "->": lambda game, node_a, node_b: game.add_connection(node_a, node_b),
-            "\->": lambda game, node_a, node_b: game.add_connection(node_a, node_b, BreakingConnection),
+            "\\->": lambda game, node_a, node_b: game.add_connection(node_a, node_b, BreakingConnection),
```

Comment:
I would make the changes to connection types in a different PR. Which is not to say that these changes are bad, they're not! They're just unrelated.